### ISSUE TITLE
How to convert .md to website.

### ISSUE THREAD

**Issue by kory-yhg** [balderdashy/www.sailsjs.org#79]
Hi, I just launched http://sailsjs.jp , the Japanese translation of the http://sailsjs.org.
I couldn't find the script for converting .md files (,which is from ja branch in the sails-docs) into website (.ejs), so I wrote it by myself but I don't like it because it's a "reinventing the wheel"
Could someone tell me how to make it with official scripts?

**Comment by mikermcneil** [comment]
Hey Kory, we're all really stoked about this, thanks!  I should have answered in this issue (my bad) but here's some considerations about our multilingual strategy and a bit of insight into how it works: https://github.com/balderdashy/sails-docs/issues/580

(I'll also back-link)

**Comment by sailsbot** [comment]
Thanks for posting, @kory-yhg.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
